### PR TITLE
Update Readme

### LINKED ISSUE #69: Update Readme
- [x] new features
- [x] link to announcement blogpost
- [x] update screenshot (use url instead of markdown link this time so it works for PyPI)
- [x] Add full image of demo pipeline
- [x] new extractor plugins
- [x] update extractor gist

### PR DESCRIPTION
**Pull request by disrupted**
Closes #69

### CODE REVIEW

**Review by philipp94831 — COMMENTED** [review]
Has kafka connect exporter been mentioned in the README? I think the other exporters are there as well, right?

File: README.md [hunk 1]
```diff
@@ -167,8 +171,10 @@ for Kafka Connect Elasticsearch connector
...
+It is possible to create your own linker and extractors in Python by implementing the `LinkingService` or `Extractor` classes. This way you can customize it to your specific setup and services. As an example we provide the [DefaultLinker](https://github.com/bakdata/streams-explorer/blob/main/backend/streams_explorer/defaultlinker.py) as `LinkingService` and [ElasticsearchSink](https://github.com/bakdata/streams-explorer/blob/main/backend/streams_explorer/core/extractor/default/elasticsearch_sink.py), [JdbcSink](https://github.com/bakdata/streams-explorer/blob/main/backend/streams_explorer/core/extractor/default/jdbc_sink.py), [S3Sink](https://github.com/bakdata/streams-explorer/blob/main/backend/streams_explorer/core/extractor/default/s3_sink.py) `Extractor` classes which are used by default.
```
Comment (VictorKuenstler) [inline README.md:180]: You could list the default extractors using bullet points to improve readability.

**Comment by disrupted** [comment]
> Has kafka connect exporter been mentioned in the README? I think the other exporters are there as well, right?

So far the other exporters haven't been mentioned in the Readme, but I think we should add them.

File: README.md (hunk 1 above)
Comment (disrupted) [reply README.md]: good idea, I changed it.

**Review by VictorKuenstler — APPROVED** [review]
LGTM

**Pull request merged**